package.json

```diff
@@ -98,6 +97,7 @@
     "fastseries": "^1.7.2",
     "find-my-way": "^1.5.0",
     "flatstr": "^1.0.5",
+    "light-my-request": "^1.0.0",
```

Comment:
Is there a good reason for keep this dependency in the production server?

**NB** this feature should not be used in your code. Only for test purpose.  The request injection works only if your plugins are registered in the same process.

IMHO this line should be moved to "dependencies" section.

Reply:
Inside the `devDependencies` we should install only what **we** need to develop Fastify itself.
Since `inject` is a public api if you install the framework with `npm i fastify —production` and we ship `light-my-request` as `devDependency`, that api won’t work.